db/schema.rb

```diff
@@ -1081,8 +1081,10 @@
     t.text "metadata"
     t.datetime "metadata_updated_at"
     t.index ["created_at", "type"], name: "index_saved_claims_on_created_at_and_type"
+    t.index ["delete_date"], name: "index_saved_claims_on_delete_date"
     t.index ["guid"], name: "index_saved_claims_on_guid", unique: true
     t.index ["id", "type"], name: "index_saved_claims_on_id_and_type"
+    t.index ["metadata", "type"], name: "index_saved_claims_on_metadata_and_type", where: "(metadata ~~ '%error%'::text)"
```

Comment:
Can we control the name of the index generated? `index_saved_claims_on_metadata_and_type` is a little misleading to someone outside Ruby as this index only has a subset of items based on the content of the `metadata` field.

Reply:
Yes, that's a good idea. I'll give it a more descriptive name.